elements/bulbs-liveblog/bulbs-liveblog-responses.js

```diff
@@ -143,8 +143,8 @@ class BulbsLiveblogResponses extends BulbsHTMLElement {
       this.append(this.responseStaging.firstElementChild);
     }
     window.picturefill();
-    if (twttr) {
-      twttr.widgets.load();
+    if (twttr) {              // eslint-disable-line no-undef
```

Comment:
gotcha. or just a wiki page somewhere

just want to adhere to conventions before I unintentionally format things differently